### PR TITLE
Also set talk_enabled in PublicViewController

### PR DESCRIPTION
**Pull request by georgehrke**
fixes undefined index

### CODE REVIEW

**Comment by tcitworld** [comment]
Can you also get the defaults values from `getAppValue` (#1816) here ?

**Comment by georgehrke** [comment]
Sure, will add that.

File: lib/Controller/PublicViewController.php [hunk 1]
```diff
@@ -97,15 +97,21 @@ public function publicIndexForEmbedding(string $token):TemplateResponse {
 	 */
 	private function publicIndex(string $token,
 								 string $renderAs):TemplateResponse {
+		$defaultInitialView = $this->config->getAppValue($this->appName, 'currentView', 'dayGridMonth');
+		$defaultShowWeekends = $this->config->getAppValue($this->appName, 'showWeekends', 'yes');
+		$defaultWeekNumbers = $this->config->getAppValue($this->appName, 'showWeekNr', 'no');
+		$defaultSkipPopover = $this->config->getAppValue($this->appName, 'skipPopover', 'no');
```
Comment (tcitworld) [inline lib/Controller/PublicViewController.php]: We had `'skip_popover' => true` as default previously, but this is fine.

Comment (georgehrke) [reply lib/Controller/PublicViewController.php]: Yeah, let me change that again.

**Pull request merged**